src/infra/services/uninstall.go

```diff
@@ -15,22 +15,48 @@ func Uninstall(name valueObject.ServiceName) error {
 	}
 
 	var packages []string
+	var supervisordConfigName string
+	nonInteractive := false
 	switch name.String() {
 	case "openlitespeed", "litespeed":
 		packages = OlsPackages
+		supervisordConfigName = "openlitespeed"
 	case "mysql", "mysqld", "maria", "mariadb", "percona", "perconadb":
 		packages = MariaDbPackages
+		supervisordConfigName = "mysql"
+		nonInteractive = true
 	case "node", "nodejs":
 		packages = NodePackages
+		supervisordConfigName = "node"
 	case "redis", "redis-server":
 		packages = RedisPackages
+		supervisordConfigName = "redis"
 	default:
 		log.Printf("ServiceNotImplemented: %s", name.String())
 		return errors.New("ServiceNotImplemented")
 	}
 
+	err = SupervisordFacade{}.Stop(name)
+	if err != nil {
+		log.Printf("UninstallServiceError: %s", err.Error())
+		return errors.New("UninstallServiceError")
+	}
+
+	err = SupervisordFacade{}.RemoveConf(supervisordConfigName)
+	if err != nil {
+		log.Printf("UninstallServiceError: %s", err.Error())
+		return errors.New("UninstallServiceError")
+	}
+
```

Comment:
Qual a diferença do name para o supervisordConfigName?

Reply:
O `name` pode ser tanto o nome do serviço, quanto um _alias_. Já o `supervisordConfigName` é o nome do serviço de fato, pois o `supervisord` só recebe o real nome do serviço no `install.go`.

Reply:
Então modifica o name dentro do switch.

Reply:
Não posso. O `name` é um VO, portanto seu dado é imutável. Se eu for recriar o VO apenas para mudar o valor dele, eu acrescentaria 4 linhas extras para cada `case` dentro do `switch`, pois eu precisaria chamar o construtor e validar se deu erro. No fim, o `name` se tornará uma variável que receberá um novo VO apenas para fazer `.String()` no `RemoveConf()` o que pode ser facilmente suprimido pelo `supervisordConfigName`.

Reply:
Enviado via Slack.